src/main/java/net/sf/jabref/logic/undo/UndoRedoEvent.java

```diff
@@ -0,0 +1,10 @@
+package net.sf.jabref.logic.undo;
+
+
+public class UndoRedoEvent extends AddUndoEvent {
```

Comment:
Another reason for inheriting was that there could be a shared function for
updating the text. I have copied that now though.

The reason that such an event is needed is that it is triggering the tool
tip changes. That is all it does.

Will push tomorrow. :-)